github/actions_secrets.go

```diff
@@ -36,20 +37,22 @@ func (s *ActionsService) GetPublicKey(ctx context.Context, owner string, repo st
 
 // Secret represents a repository action secret.
 type Secret struct {
-	Name      *string    `json:"name,omitempty"`
-	CreatedAt *Timestamp `json:"created_at,omitempty"`
-	UpdatedAt *Timestamp `json:"updated_at,omitempty"`
+	Name      *string    `json:"name"`
```

Comment:
Since you removed the `omitempty` tag modifiers for these three values (I was only talking about the structs below, but this is actually fine and good), and since the GitHub documentation makes it appear that these value are not optional and will always be returned, let's please change these three variables from pointers to values.

Reply:
Done